src/gestures-hooks.tsx

```diff
@@ -151,7 +129,36 @@ class GestureActions {
     private setGestureState: (gestureData: GestureContextState) => void,
     private status: MlStatus,
     private setStatus: (status: MlStatus) => void
-  ) {}
+  ) {
+    // Initialize with at least one gesture for walkthrough.
+    if (!this.gestureState.data.length) {
+      this.setGestures([], false);
```

Comment:
I was initially confused how setting the gestures as an empty array was going to ensure at least one gesture for walkthrough. Took me a bit more longer to realise that the "one gesture for walkthrough" was being set in the  `setGestures` method. Nitpicking, but maybe this would make it more explicit?

```suggestion
      this.setGestureState({ data: [this.generateNewGesture()] });
```